gt/main.c

```diff
@@ -170,32 +177,72 @@ gt_parse_incoming_pkt(struct rte_mbuf *pkt, struct gt_packet_headers *info)
 
 		info->inner_ip_ver = ETHER_TYPE_IPv6;
 		info->l4_hdr = (uint8_t *)inner_ipv6_hdr + l4_offset;
+		info->frag_hdr =
```

Comment:
I cannot comment on the line above, but we should not require 4 bytes from the L4 header for the IPv6 case as we do earlier for the IPv4 in this function, namely, `gt_parse_incoming_pkt()`.